suite/tests/CMakeLists.txt

```diff
@@ -2223,7 +2223,8 @@ if (CLIENT_INTERFACE)
   use_DynamoRIO_extension(client.drxmgr-test.dll drx)
   use_DynamoRIO_extension(client.drxmgr-test.dll drreg)
 
-  tobuild_ci(client.low_on_memory client-interface/low_on_memory.c "" "-vm_size 4M" "")
+  tobuild_ci(client.low_on_memory client-interface/low_on_memory.c ""
+      "-enable_reset -reset_at_vmm_percent_free_limit 10 -vm_size 4M" "")
```

Comment:
If this is how it has to be, then please add a comment explaining why this parameter is here.  But, it seems like it shouldn't be necessary: -enable_reset should turn this on.  I'm looking in optionsx.h: it seems like we shouldn't need to have all those IF_X86_ELSE splits for the defaults, right?  DISABLE_RESET should take care of clearing those.  I.e., I'm proposing removing all of those: unless I'm missing something, they should still be 0 on arm by default.

Reply:
Yes, I had a similar reading of the header file. I'll include a comment regarding the param for this PR, and raise an issue regarding IF_X86_ELSE since it is unrelated to this code.

Reply:
Sure.

Reply:
Done. I also added the issue as discussed.

Reply:
@derekbruening Do I have to go ahead to merge please?

Reply:
Yes.